internal/services/notification/notification_test.go

```diff
@@ -709,6 +709,56 @@ func TestDeliveryStatusFromStringSlice(t *testing.T) {
 	}
 }
 
+func TestRunWebhooksCleaner(t *testing.T) {
+	t.Parallel()
+
+	dir := t.TempDir()
+	log := testutil.NewLogger(t)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	ns := setupNotificationService(ctx, t, log, dir)
```

Comment:
It will be better to test with some not expired webhooks. See comment https://github.com/agola-io/agola/pull/457/files#diff-28c811e42db92fc3fe4788b59c6137497529a96822247f6a60626c8ecc71af49